firebase-inappmessaging-display/src/main/java/com/google/firebase/inappmessaging/display/FirebaseInAppMessagingDisplay.java

```diff
@@ -292,6 +292,12 @@ public void run() {
   // it is safe to ignore this warning
   @SuppressLint("ClickableViewAccessibility")
   private void inflateBinding(final Activity activity, final BindingWrapper bindingWrapper) {
+    if (inAppMessage == null) {
```

Comment:
Just want to be sure that we are not masking any deeper bug with this check. Is it OK to do nothing in these cases? what would be the user experience in this case? e.g. a message could be lost, several messages, etc.

Reply:
I've already have my analysis mentioned here : https://github.com/firebase/firebase-android-sdk/issues/4214#issuecomment-1433779590

So if the race condition happens, we will not be presenting the inappmessage during that event and will be shown when then next same event happens. If we get significant reports on issues related with this, we will look towards a long term solution.